.circleci/config.yml

```diff
@@ -151,85 +151,70 @@ workflows:
       - build
       - hold:
           type: approval
+      - master:
+          requires:
+            - build
           filters:
             branches:
-              ignore: master
+              only: master
+      - pr:
           requires:
-          - build
+            - build
+            - hold
+          filters:
+            branches:
+              ignore: master
 
 
 
-      - e2eTestCurPRClusterState:
-          requires:
-          - hold
-      - e2eTestWIPPRClusterState:
-          requires:
-          - hold
       - e2eTestCurMasterClusterState:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
       - e2eTestWIPMasterClusterState:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
-
-
+            - master
+      - e2eTestCurPRClusterState:
+          requires:
+            - pr
+      - e2eTestWIPPRClusterState:
+          requires:
+            - pr
 
 
 
-      - e2eTestCurPRDraining:
-          requires:
-          - hold
-      - e2eTestWIPPRDraining:
-          requires:
-          - hold
       - e2eTestCurMasterDraining:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
       - e2eTestWIPMasterDraining:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
+      - e2eTestCurPRDraining:
+          requires:
+            - pr
+      - e2eTestWIPPRDraining:
+          requires:
+            - pr
 
 
 
-      - e2eTestCurPRScaling:
-          requires:
-          - hold
-      - e2eTestWIPPRScaling:
-          requires:
-          - hold
       - e2eTestCurMasterScaling:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
       - e2eTestWIPMasterScaling:
-          filters:
-            branches:
-              only: master
           requires:
-          - build
+            - master
+      - e2eTestCurPRScaling:
+          requires:
+            - pr
+      - e2eTestWIPPRScaling:
+          requires:
+            - pr
 
 
 
       - e2eTestCurMasterUpdate:
```

Comment:
@xh3b4sd should we run these in WIP as well?

Reply:
There should be a comment on the other definition that this makes no sense to be run in WIP. The test takes the current and wip version anyway in order to perform the upgrade. That is why the upgrade test is only there once and not twice.

Reply:
To clarify. "There should be a comment" means the comment should be there but it disappeared or it should be added?